internal/pipeline/testdata/simplepackage/.expected/config.yaml

```diff
@@ -0,0 +1,2 @@
+exitCode: 0
+network: false
```

Comment:
Do we need to check idempotence for pipeline? @frankfarzan

Reply:
Spoke briefly with @droot about this and he said it is not a requirement for pipeline run because it is a property of the kpt functions